### DISCUSSION TITLE
Please add shortcut command for suggestion

### DISCUSSION THREAD

**Discussion by kythuatwebso** [community/community#7172] [Copilot Conversations] [answered]
Hello,

Feature auto suggestion something annoying, So I think it's better to call with a shortcut.

![image](https://user-images.githubusercontent.com/57938377/140002125-9657d2cc-46b0-4ad7-9f57-a2cb29584136.png)

Thanks!

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
Please check our documentation about available commands, and keyboard shortcuts here - https://github.com/github/copilot-docs/blob/main/docs/visualstudiocode/configuring.md

**Reply by kythuatwebso** [reply]
Please let me know,
This setting is applicable to both vscode editor or to **copilot** extension only?

I am not found setting for suggestion for **copilot** shortcut

My mean is setting for only copilot extension.

**Comment by asgerhallas** [comment]
I can not get this to work. If I turn of inline suggestions, they don't show up when I use the keyboard shortcut either. How can I turn off automatic inline suggestions, and only have them shown when I request it by keyboard shortcut?

**Comment by asgerhallas** [comment]
Also the links are broken. The docs can be found here https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-in-visual-studio-code
